src/BuildAnalyzer/BuildAnalyzer.h

```diff
@@ -36,10 +44,10 @@ class BuildAnalyzer
     inline bool IsAnalysisPerformed() const { return m_analysisPerformed; }
 
     bool Analyze();
-    bool ExportFunctionCompilationsData(const std::string& path) const;
-    bool ExportFileInclusionTimesData(const std::string& path) const;
+    bool ExportFunctionCompilationsData(const std::string& path, TimeDisplayEnum timeDisplay) const;
+    bool ExportFileInclusionTimesData(const std::string& path, TimeDisplayEnum timeDisplay) const;
     bool ExportFileInclusionGraph(const std::string& path) const;
-    bool ExportFileCompilationsData(const std::string& path) const;
+    bool ExportFileCompilationsData(const std::string& path, TimeDisplayEnum) const;
```

Comment:
Can you please add a name to this new parameter (i.e. `timeDisplay`)?

Reply:
this should be done